### DISCUSSION TITLE
Response header is missing content-type

### DISCUSSION THREAD

**Discussion by dmali-dev** [emmett-framework/granian#405] [Q&A]
Hi team,

I am using sanic server in my code for creating a app and routing purpose, also setting a content-type in header using sanic. I am running my app using granian. In the curl response I can not see content-type when I tried to run it by granian. Same code works fine with uvicorn or sanic command.

< HTTP/1.1 100 Continue
* We are completely uploaded and fine
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< server: granian
< content-length: 120
< date: Wed, 25 Sep 2024 06:10:51 GMT

**Comment by gi0baro** [comment]
@dmali92 what's the value you're setting for the content type?

**Reply by dmali-dev** [reply]
Hello @gi0baro, I am setting content type as application/json using sanic framework.

**Reply by gi0baro** [reply]
Can you provide a MRE for this?
